rules/import/on.js

```diff
@@ -46,6 +46,6 @@ module.exports = {
     //Enforce a newline after import statements
     "import/newline-after-import": 0,
     //Prefer a default export if module exports a single name
-    "import/prefer-default-export": 0,
+    "import/prefer-default-export": 0
```

Comment:
Good catch! We still have a travis config from the fork but we need to enable it. Just added that to the issues
https://github.com/FormidableLabs/eslint-config-formidable/blob/master/.travis.yml